node/gulpfile.js

```diff
@@ -34,8 +34,32 @@ gulp.task('clean', function (done) {
 	return del([buildRoot], done);
 });
 
-gulp.task('copy:manifest', ['clean'], function () {
-	return gulp.src(['package.json', 'lib.json', '../LICENSE', '../README.md'])
+gulp.task('loc:generate', ['clean'], function() {
+    // Build the content for the en-US resjson file.
+    var lib = require('./lib.json');
+    var strPath = './Strings/resources.resjson/en-US';
```

Comment:
path.join('.', 'Strings', 'resources.resjson', 'en-US') is a better node way.  But you can change after PR - no reason to block anything